src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -548,19 +641,25 @@ static NamedArgsAndClosure parseArgs(Object arg, StepDescriptor d) {
      * @param soleArgumentKey
      *      If the context in which this method call happens allow implicit sole default argument, specify its name.
      *      If null, the call must be with names arguments.
+     * @param interpolatedStrings
+     *      The collection of interpolated Groovy strings.
      */
-    static NamedArgsAndClosure parseArgs(Object arg, boolean expectsBlock, String soleArgumentKey, boolean singleRequiredArg) {
-        if (arg instanceof NamedArgsAndClosure)
+    static NamedArgsAndClosure parseArgs(Object arg, boolean expectsBlock, String soleArgumentKey, boolean singleRequiredArg, @Nonnull Set<String> interpolatedStrings) {
+        if (arg instanceof NamedArgsAndClosure) {
```

Comment:
nit: You could revert the whitespace changes and added braces in this method to keep the diff a little easier to read (even though the new style is generally recommended)